### PR TITLE
Feed Scroll Performance

### PR DESCRIPTION
**Pull request by TyagiLikeMinds**
## Description

**Features**

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fix for breaking changes)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [ ] Assignee added
- [ ] Reviewer added
- [ ] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [ ] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [ ] Label added
- [ ] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/PostDetail/LMFeedBasePostDetailViewModel.swift [hunk 1]
```diff
@@ -26,7 +26,7 @@ public class LMFeedBasePostDetailViewModel {
         self.postID = postID
         self.commentList = []
         self.currentPage = 1
-        self.pageSize = 10
+        self.pageSize = 20
```
Comment (devansh-mohata) [inline lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/PostDetail/LMFeedBasePostDetailViewModel.swift:29]: Was it intentional or was for testing purpose ?

File: lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/PostList/LMFeedBasePostListViewModel.swift [hunk 2]
```diff
@@ -34,7 +34,7 @@ public class LMFeedBasePostListViewModel {
     
     init(delegate: LMFeedBasePostListViewModelProtocol) {
         self.currentPage = 1
-        self.pageSize = 10
+        self.pageSize = 20
```
Comment (devansh-mohata) [inline lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/PostList/LMFeedBasePostListViewModel.swift:37]: Was it intentional or was for testing purpose ?

File: lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/SearchPosts/LMFeedBaseSearchPostViewModel.swift [hunk 3]
```diff
@@ -40,7 +40,7 @@ public class LMFeedBaseSearchPostViewModel {
         self.delegate = delegate
         self.postList = []
         self.page = 1
-        self.pageSize = 10
+        self.pageSize = 20
```
Comment (devansh-mohata) [inline lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/SearchPosts/LMFeedBaseSearchPostViewModel.swift:43]: Was it intentional or was for testing purpose ?

File: lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/TaggingListView/LMFeedTaggingListViewModel.swift [hunk 4]
```diff
@@ -29,7 +29,7 @@ public final class LMFeedTaggingListViewModel {
     // MARK: Init
     init(delegate: LMFeedTaggingListViewModelProtocol?) {
         self.currentPage = 1
-        self.pageSize = 10
+        self.pageSize = 20
```
Comment (devansh-mohata) [inline lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/TaggingListView/LMFeedTaggingListViewModel.swift:32]: Was it intentional or was for testing purpose ?

File: lm-feedUI-iOS/lm-feedUI-iOS/Source/Classes/Initials/LMTableView.swift [hunk 5]
```diff
@@ -85,27 +85,32 @@ public class LMTableView: UITableView {
...
     open func reloadTable(for index: IndexPath? = nil) {
-        UIView.performWithoutAnimation { [weak self] in
-            if let index {
-                if index.row == NSNotFound {
-                    
-                    self?.reloadSections(IndexSet(integer: index.section), with: .none)
-                } else {
-                    self?.reloadRows(at: [index], with: .none)
-                }
+        // UIView.performWithoutAnimation { [weak self] in
```
Comment (devansh-mohata) [inline lm-feedUI-iOS/lm-feedUI-iOS/Source/Classes/Initials/LMTableView.swift:88]: remove comments if not needed

**Review by devansh-mohata — APPROVED** [review]

File: lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/TaggingListView/LMFeedTaggingListViewModel.swift (hunk 4 above)
Comment (TyagiLikeMinds) [reply lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/TaggingListView/LMFeedTaggingListViewModel.swift:32]: It was intentional

File: lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/SearchPosts/LMFeedBaseSearchPostViewModel.swift (hunk 3 above)
Comment (TyagiLikeMinds) [reply lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/SearchPosts/LMFeedBaseSearchPostViewModel.swift:43]: It was intentional

File: lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/PostList/LMFeedBasePostListViewModel.swift (hunk 2 above)
Comment (TyagiLikeMinds) [reply lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/PostList/LMFeedBasePostListViewModel.swift:37]: It was intentional

File: lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/PostDetail/LMFeedBasePostDetailViewModel.swift (hunk 1 above)
Comment (TyagiLikeMinds) [reply lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/PostDetail/LMFeedBasePostDetailViewModel.swift:29]: It was intentional

**Pull request merged**